packages/deployer/src/deployer.ts

```diff
@@ -36,7 +36,7 @@ export class Deployer {
      */
     public async deployAsync(contractName: string, args: any[] = []): Promise<Web3.ContractInstance> {
         const contractArtifact: ContractArtifact = this._loadContractArtifactIfExists(contractName);
-        const contractData: ContractData = this._getContractDataFromArtifactIfExists(contractArtifact);
```

Comment:
should we rename the methods as well? `this._getContractNetworkDataFromArtifactIfExists`